packages/block-library/src/group/edit.js

```diff
@@ -43,6 +65,19 @@ function GroupEdit( {
 						},
 					] }
 				/>
+
+				<PanelBody title={ __( 'Spacing' ) }>
+					<ResponsiveBlockControl
```

Comment:
I guess we should mark the component as experimental for some time.

Reply:
Definitely agreed! Done.